lib/publiccloud/utils.pm

```diff
@@ -241,7 +241,12 @@ sub gcloud_install {
     my $dir = $args{dir} || 'google-cloud-sdk';
     my $timeout = $args{timeout} || 700;
 
-    zypper_call("in curl tar gzip", $timeout);
+    # WARNING:  Python 3.6.x is no longer officially supported by the Google Cloud CLI
+    # and may not function correctly. Please use Python version 3.8 and up.
+    my @pkgs = qw(curl tar gzip);
```

Comment:
Kind question, why in place of below, not used a simpler update like:
```suggestion +244,+246:
    zypper_call("in curl tar gzip python39", $timeout);
```

Reply:
mostlikely a condition will be needed in the python version selection

Reply:
Moreover, I noted in tests 15-4 and 15-5 MRU has python3 (last one):
`MRU_ADDONS: we,sdk,base,script,serverapp,desktop,legacy,contm,python3`
while in 15-3, python2:
`MRU_ADDONS: sdk,base,script,serverapp,desktop,legacy,contm,pcm,python2`
inherited by SCC_ADDONS.

Should those ones be update too? Or set python39 there?

Reply:
those are repos/modules, AFAICT we do not need to touch those lines

Reply:
> mostlikely a condition will be needed in the python version selection
Sorry, do you mean, in a future an 'if' will be added here, about the version ?

Reply:
yes, please let me run the other VRs, the clonning process takes too long atm